hawc/apps/assessment/admin.py

```diff
@@ -33,12 +34,30 @@ class AssessmentAdmin(admin.ModelAdmin):
         "reviewers__last_name",
     )
 
-    actions = (bust_cache,)
+    actions = (bust_cache, "delete_orphan_tags")
 
     def queryset(self, request):
         qs = super().queryset(request)
         return qs.prefetch_related("project_manager", "team_members", "reviewers")
 
+    def delete_orphan_tags(self, request, queryset):
```

Comment:
most of this logic should be in in a lit manager object (for Reference or ReferenceTag); keep the biz logic outside the admin